### ISSUE TITLE
[FEATURE] [Linter] unmatched quote and bracket does not trigger error

### ISSUE THREAD

**Issue by alphatownsman** [djlint/djLint#705]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: e.g. ubuntu macOS 13.4
 - Python Version 3.11.4
 - djLint Version 1.31.1
 - template language: django

## Issue
see below for example

## How To Reproduce
```
➜  cat t.html
{% url 'journal:user_tag_list" user.url }}
➜  djlint t.html
Linting 1/1 files ━━━━━━━━━━━━━━━━━━━━━━━ 00:00
Linted 1 file, found 0 errors.
```

## Contents of .djlintrc/pyproject.toml [tool.djlint]
none

**Comment by christopherpickering** [comment]
hey @alphatownsman 

1. Rule T027 is not triggered because there are no complete template blocks in your code
2. There is no existing rule for orphan template tags.

I will leave this open as a feature request for #2. Thanks!
